src/Config/ObjectModel/DataSourceFiles.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    /// <summary>
+    /// DataSourceFiles is a record that contains a list of files defining the runtimeConfigs for multi-db scenario.
+    /// </summary>
+    /// <param name="SourceFiles"></param>
+    public record DataSourceFiles(IEnumerable<string>? SourceFiles = null);
```

Comment:
Would this class be ever referenced for single-db scenario? It maybe then worthwhile to mention that in single-db scenarios, this list would be null.